### PR TITLE
[TACHYON-568] Change private member variable to private static final in PairTest.java

### PR DESCRIPTION
**Pull request by debugger87**
https://tachyon.atlassian.net/browse/TACHYON-568

### CODE REVIEW

**Comment by apc999** [comment]
Jekins, add to whitelist

**Comment by haoyuan** [comment]
ok to test

**Comment by apc999** [comment]
+1 LGTM. Thanks for the patch!

File: common/src/test/java/tachyon/PairTest.java [hunk 1]
```diff
@@ -22,22 +22,22 @@
...
-  private Long mLongType = 9887L;
-  private Double mDoubleType = 3.14159;
-  private Boolean mBoolType = false;
-  private Character mCharType = 'a';
+  private static final String STRTYPE = "test";
+  private static final Integer INTEGERTYPE = 7883;
+  private static final Long LONGTYPE = 9887L;
+  private static final Double DOUBLETYPE = 3.14159;
+  private static final Boolean BOOLTYPE = false;
+  private static final Character CHARTYPE = 'a';
```
Comment (calvinjia) [inline common/src/test/java/tachyon/PairTest.java]: Could you separate the words with `_`? (`STR_TYPE`)

Comment (debugger87) [inline common/src/test/java/tachyon/PairTest.java]: Yeah, I will separate them with `_`.

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by haoyuan** [comment]
Thanks!

**Pull request merged**